### DISCUSSION TITLE
vermelding "diepte lijkt ongewoon diep"

### DISCUSSION THREAD

**Discussion by jro73** [Nelos-VZW/dives-2#19] [Q&A]
Bij "aanmaak duik activiteit" verschijnt bij ingave diepte van meer dan 100m een waarschuwing "diepte lijkt ongewoon diep". Zou dit niet beter zijn vanaf 60m (cfr mailing Nelos van juni dit jaar ivm diep duiken) of als de toegestane diepte conform kwalificatie wordt overschreden?

**Comment by ndsmyter** [comment]
We hebben wel met dat idee gespeeld om die melding al vanaf 60m te tonen. Maar het probleem hiermee vind ik, dat er wel ook effectief duiken dieper dan 60m mogelijk en normaal zijn in sommige situaties. Dat is dan niet met lucht, maar wel met andere gasmengsels.

Dus daarom zijn we maar gegaan om de limiet op 100m te leggen.
